### DISCUSSION TITLE
Feature request: defaults

### DISCUSSION THREAD

**Discussion by cerebrate** [jseidl/magic-areas#181] [Ideas / Feature Requests]
I'd like there to be a way to set default parameter values for all areas that don't _specifically_ have that parameter defined. There are a fair few parameters (from enabled features through presence event timeouts, minimum number of entities to create aggregates, even night-state entities, etc., that I want to be the same for every area (or every interior/exterior area) except when specifically changed, and it would be nice to have a place to change all the not-set-explicitlies in one go.

**Comment by cerebrate** [comment]
As a side note: this might be a bit of a breaking change now, but since I mentioned minimum number of entities to create aggregates, I might also suggest that the system default for this one might be better as 1 than 2?

(Having had occasion recently to redesign my configuration after a server crash, one of the first changes I made in every area was to set this to 1, on the grounds that that way, I would _never again_ have to go through all my templates, automations, etc. fixing up the references just because I added a second temperature sensor to a room... 😄 )

**Comment by jseidl** [comment]
That's a really good idea. Some kind of global overrides, but I'm not sure how easy will be to implement. Will keep on the list.

Changing defaults isn't quite breaking as once you've created the area in any form (either by installing it first time or by editing it), it will save the current configuration (using the current default whenever it was created/updated). I also like to use these as 1 because I like to use aggregators directly as well. This was set to 2 for the expectations (Why there's a group with a single entity?). I try to follow a less-spammy approach to not freak users out when they first install.
